### PR TITLE
needsFinalVote sort option

### PR DESCRIPTION
**Pull request by Raemon**
Previously we had needsReview as the main sorting option for the review phase. This creates the default sorting option for the voting phase, which is "posts I haven't voted on, followed by posts I have voted on sorted by vote-weight".

### CODE REVIEW

File: packages/lesswrong/lib/collections/posts/views.ts [hunk 1]
```diff
@@ -1343,8 +1343,8 @@ ensureIndex(Posts,
 Posts.addView("reviewFinalVoting", (terms: PostsViewTerms) => {
   return {
     selector: {
-      reviewCount: { $gte: 1},
-      positiveReviewVoteCount: { $gte: 1 },
+      reviewCount: { $gte: 1 },
+      positiveReviewVoteCount: { $gte: 1 }, // TODO: Ray thinks next year this should change to "has at least 4 points"
```
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/posts/views.ts:1347]: Why not this year?

Comment (Raemon) [reply packages/lesswrong/lib/collections/posts/views.ts:1347]: I went back and forth a bit, and decided the benefits of filtering them out didn't outweigh the integrity-hit to changing the requirements last minute. (I'm not that confident in that, but, currently think the effect size is smallish either way)

**Comment by Raemon** [comment]
I've updated this so that the needsVote sorting includes randomization. I tested that different users saw the posts in a different order, and that they both saw their voted-posts in proper order. Tried changing the sort to some other random ways and it seemed to work.

File: packages/lesswrong/lib/collections/posts/views.ts (hunk 1 above)
Comment (jpaddison3) [reply packages/lesswrong/lib/collections/posts/views.ts:1347]: Yeah, makes sense

**Review by jpaddison3 — COMMENTED** [review]
[Post-merge-review] LGTM

**Pull request merged**